File: chap11/run-model11-7.R
library(cmdstanr)

d <- read.csv('input/data-switch.csv')
T <- nrow(d)
data <- list(T=T, Y=d$Y, B=0.2)

model <- cmdstan_model('model/model11-7.stan')
fit <- model$sample(data=data, seed=123, parallel_chains=4,
                    max_treedepth=15)
fit$save_object(file='output/result-model11-7.RDS')

File: chap11/run-model11-7.py
import pandas
import cmdstanpy

d = pandas.read_csv('input/data-switch.csv')
T = len(d)
data = {'T':T, 'Y':d.Y, 'B':0.2}

model = cmdstanpy.CmdStanModel(stan_file='model/model11-7.stan')
fit = model.sample(data=data, seed=123, parallel_chains=4, max_treedepth=15)
fit.save_csvfiles('output/result-model11-7')
